### PR TITLE
Automate config rule updates

### PR DESCRIPTION
**Pull request by Fozruk**

### CODE REVIEW

**Comment by grothja** [comment]
@duraikkannuv2 could you work on testing this out in our Test Org?  Feel free to talk to Carl directly about anything related to this as well for testing, structure, etc.

File: examples/complete/main.tf [hunk 1]
```diff
@@ -29,15 +29,15 @@ module "managed_rules" {
...
-    allowVersionUpgrade = "true"
+    allowVersionUpgrade = true
   }
 
   rule_overrides = {
     acm-certificate-expiration-check = {
       description = "Checks if AWS Certificate Manager Certificates in your account..."
 
       input_parameters = {
-        daysToExpiration = tostring(var.acm_certificate_expiration_check)
```
Comment (Fozruk) [inline examples/complete/main.tf:40]: The same treatment as above was made here. The `daysToExpiration` parameter in this example expects a `number` that will be converted to a string later.

File: modules/organization/main.tf [hunk 2]
```diff
@@ -8,8 +8,12 @@ resource "aws_config_organization_managed_rule" "rule" {
   resource_types_scope = try(each.value["resource_types_scope"], [])
 
   input_parameters = (
+    # AWS Config expects all values as strings. This list comprehension
+    # removes optional parameter attributes whose value is 'null'.
     try(jsonencode(each.value["input_parameters"]), null) != "null" ?
-    try(jsonencode(each.value["input_parameters"]), null) :
```
Comment (Fozruk) [inline modules/organization/main.tf:12]: Same as above except for the `organization` module.

**Comment by duraikkannuv2** [comment]
> @duraikkannuv2 could you work on testing this out in our Test Org? Feel free to talk to Carl directly about anything related to this as well for testing, structure, etc.

Sure. I will do.

**Pull request merged**